packages/@creature-chess/ui/src/cardShop/cardShop.tsx

```diff
@@ -14,6 +14,7 @@ type Props = {
 	onReroll?: () => void;
 	onToggleLock?: () => void;
 	onBuy?: (index: number) => void;
+	devMode?: boolean;
```

Comment:
This is to prevent the current card popping up in the developers tools, which I've done to keep the ui looking trim. Not sure if this is the best solution though?

Reply:
gonna undo this - the UI package should have no idea where it will be used

Reply:
in fact, I'll keep it, but I will rename it to `showSelectedCard`